images/server/Dockerfile.fedora

```diff
@@ -1,13 +1,13 @@
 FROM quay.io/samba.org/sambacc:latest AS builder
-ARG SAMBACC_VER=f6480c5861a56ef9d1ebb965aebb14732ec2690c
+ARG SAMBACC_VER=fd747527ed86d01821c4a3b72f1c4f893bb1ca36
 ARG SAMBACC_REPO=https://github.com/samba-in-kubernetes/sambacc
 
 # the changeset hash on the next line ensures we get a specifc
 # version of sambacc. When sambacc actually gets tagged, it should
 # be changed to use the tag.
 RUN /usr/local/bin/build.sh ${SAMBACC_VER} ${SAMBACC_REPO}
 
-FROM fedora
+FROM registry.fedoraproject.org/fedora:34
```

Comment:
[samba-4.15.2-3.fc35](https://bodhi.fedoraproject.org/updates/FEDORA-2021-12af2614da), which have fixes for `smbclient` anonymous login issues, should be available in stable repositories soon(mostly eod). So let's get to 35 while we are at it.

Reply:
Sure, but I want to keep things moving forward in a step-by-step manner. This just codifies what we already had.

Reply:
Ok.